packages/braid-design-system/src/lib/utils/source.macro.js

```diff
@@ -1,7 +1,8 @@
 const { createMacro } = require('babel-plugin-macros');
 const { default: generate } = require('@babel/generator');
 
-module.exports = createMacro(({ babel: { types: t }, references }) => {
+exports.__esModule = true;
+exports.default = createMacro(({ babel: { types: t }, references }) => {
```

Comment:
I'm having some trouble with this bad boy. It either works in Crackle or in Webpack,  but not both 😩

Reply:
Solved in [`4f9edc6` (#1223)](https://github.com/seek-oss/braid-design-system/pull/1223/commits/4f9edc66ac4b9185e0035ab98d86cbd486f2acd3)